develop/schemadoc/PackageManifest/Manifest.md

```diff
@@ -11,9 +11,10 @@ Manifest file of a package.
 |Name|Occurrences|Description|
 |--- |--- |--- |
 |***Sequence***|||
-|&nbsp;&nbsp;[Name](xref:Manifest.Name)||Specifies the name of the package.|
-|&nbsp;&nbsp;[Version](xref:Manifest.Version)||Specifies the version of the package.|
-|&nbsp;&nbsp;[Content](xref:Manifest.Content)||Specifies the content of the package.|
-|&nbsp;&nbsp;[DeleteFiles](xref:Manifest.DeleteFiles)|[0, 1]|Specifies files to delete.|
-|&nbsp;&nbsp;[DeploymentOptions](xref:Manifest.DeploymentOptions)|[0, 1]|Specifies options for registration and deployment of the package.|
-|&nbsp;&nbsp;[VersionHistory](xref:Manifest.VersionHistory)|[0, 1]|Contains an overview of the version history of this package.|
+|[Name](xref:Manifest.Name)||Specifies the name of the package.|
+|[Version](xref:Manifest.Version)||Specifies the version of the package.|
+|[Content](xref:Manifest.Content)||Specifies the content of the package.|
+<!-- |[DeleteFiles](xref:Manifest.DeleteFiles)|[0, 1]|Specifies files to delete.| -->
```

Comment:
table is not rendering correctly anymore, probably because of the commented out line. Instead of commenting it out, remove it.